tests/integration/targets/test_azure_manage_security_group/tasks/main.yml

```diff
@@ -6,92 +6,101 @@
   register: sg_info
   failed_when: sg_info.securitygroups | length > 0
 
-# Test: Create Security Group
-- name: Create security group with required rules
-  ansible.builtin.include_role:
-    name: cloud.azure_ops.azure_manage_security_group
-  vars:
-    azure_manage_security_group_operation: create
-    azure_manage_security_group_resource_group: "{{ resource_group }}"
-    azure_manage_security_group_security_group:
-      name: "{{ security_group_name }}"
-      rules: "{{ security_group_rules }}"
+- name: Test role cloud.azure_ops.azure_manage_security_group
```

Comment:
I think the name before was good
This task creates the security group
Most of the tasks in this file test "cloud.azure_ops.azure_manage_security_group", its not only this task

Reply:
I added a 'block' section which includes all previous tasks. This name for block, name of task didn't change. This structure is needed to add an 'always' section.